frontend/whiteboard-app/src/components/WhiteboardPage.tsx

```diff
@@ -4,7 +4,7 @@ import useNoteRequest from "../hooks/useNoteRequest";
 const WhiteboardPage = () => {
     const {notes} = useNoteRequest();
     return  (
-        <>
+        <div className="whiteBoardPageParent">
```

Comment:
es wäre sinnvoll, wenn du eine extra whiteboard komponente erstellst, und diese in auf die seite einbindest. sieh dir nochmal das styling beispiel an, du hast hier nur die page selbst gestyled, im ticket geht es aber um das eigentliche whiteboard

Reply:
Hi Anke , ok alles klar  ich habe  jetzt eine neue Komponent erzeugt , und diese  habe ich  ihn mit WhiteboardPage Komponent  verbunden